packages/Webkul/Shop/src/Resources/views/guest/compare/compare-products.blade.php

```diff
@@ -28,7 +28,7 @@
         }
```

Comment:
Remove internal css

Reply:
done

Reply:
execute **npm run prod** command in shop package

Reply:
excuted the command and pushed the code